### ISSUE TITLE
SpaContentPlaceHolder don't update ContentPlaceHolder

### ISSUE THREAD

**Issue by martindybal** [riganti/dotvvm#718]
I have a page with one `SpaContentPlaceHolder` and one `ContentPlaceHolder` I need update both of them on SPA Redirect. I found a workaround and I set `PostBack.Update="true"` to a parent of `ContentPlaceHolder`. But I don't need refresh on each postback. I'm thinking about new attached property, something like `SPARedirect.Update=true`

**Comment by tomasherceg** [comment]
We should think about some conditional way of updating parts of the page.

Right now, we have `PostBack.Update` which updates the content on all postbacks, and also on SPA navigation.

I am thinking about `PostBack.UpdateOnSpaNavigate` which would update the content only on SPA navigation. 

Also, we might introduce something like "update groups" - something like `PostBack.UpdateGroups="my-group, second-group"` and then calling something `Context.RefreshPostBackUpdateGroup("my-group");` during the postback. 
That would allow us to refresh only some parts of the page.

However, I'm not sure about the demand for this feature.

**Comment by exyi** [comment]
I don't want to introduce more complications, using the postback update groups would be pretty complicated and just creates hidden dependencies :/ I think that updating all `ContentPlaceHolders` by default (or allowing multiple `SpaContentPlaceholder`s) would be much easier to everyone.

**Comment by tomasherceg** [comment]
I agree that postback update groups are probably too much. Maybe we should allow more `SpaContentPlaceHolder`s on one page - it's just a regular `ContentPlaceHolder` with PostBack.Update=true anyway. And as far as I remember, the only reason for having just one on the page is to be able to identify the master page uniquely to recognize if a redirect is needed. 
This can be done differently.
